test/ci/buildkite/ngtf-cpu_ubuntu-bin-build.yaml

```diff
@@ -12,6 +12,7 @@
 
   - command: |
       source /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/venv/bin/activate 
+      python3 build_ngtf.py --use_tensorflow_from_location /localdisk/buildkite-agent/prebuilt_tensorflow_1_15_0
```

Comment:
We run "build_ngtf.py" twice in this yaml? One with use_tensorflow_from_location one with use_prebuilt_tensorflow?

Reply:
Let me check

Reply:
We shouldn't need this line